nip47/controllers/cancel_hold_invoice_controller.go

```diff
@@ -0,0 +1,72 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/getAlby/hub/constants"
+	"github.com/getAlby/hub/logger"
+	"github.com/getAlby/hub/nip47/models"
+	"github.com/nbd-wtf/go-nostr"
+	"github.com/sirupsen/logrus"
+)
+
+type cancelHoldInvoiceParams struct {
+	PaymentHash string `json:"payment_hash"`
+}
+type cancelHoldInvoiceResponse struct{}
+
+func (controller *nip47Controller) HandleCancelHoldInvoiceEvent(ctx context.Context, nip47Request *models.Request, requestEventId uint, appId uint, publishResponse func(*models.Response, nostr.Tags)) {
+	if nip47Request == nil {
```

Comment:
this can't be null, can you remove it from the new controller methods and make sure they follow the same conventions as the existing methods?